fluent-fallback/tests/localization_test.rs

```diff
@@ -488,3 +497,32 @@ async fn localization_handle_state_changes_mid_async() {
 
     bundles.format_value("key", None, &mut errors).await;
 }
+
+#[test]
+fn localization_duplicate_resources() {
+    let resource_ids: Vec<ResourceId> =
+        vec!["test.ftl".into(), "test2.ftl".into(), "test2.ftl".into()];
+    let locales = Locales::new(vec![langid!("pl"), langid!("en-US")]);
+    let res_mgr = ResourceManager;
+    let mut errors = vec![];
+
+    let loc = Localization::with_env(resource_ids, true, locales, res_mgr);
+    let bundles = loc.bundles();
+
+    let value = bundles
+        .format_value_sync("hello-world", None, &mut errors)
+        .unwrap();
+    assert_eq!(value, Some(Cow::Borrowed("Hello World [pl]")));
+
+    let value = bundles
+        .format_value_sync("missing-message", None, &mut errors)
+        .unwrap();
+    assert_eq!(value, None);
+
+    let value = bundles
+        .format_value_sync("hello-world-3", None, &mut errors)
+        .unwrap();
+    assert_eq!(value, Some(Cow::Borrowed("Hello World 3 [en]")));
+
+    assert_eq!(errors.len(), 4);
```

Comment:
Suggestion: Can we assert on the contents of the errors here? I'm not clear on what errors we are expecting. From the description in Bug 1755216 it sounds like we want to suppress any errors from adding `test2.ftl` a second time, but allow other errors through.

Reply:
I was lazy and copied and pasted a pre-existing test. I simplified it so that there are no errors.